tests/integration/fuzz/MerkleVCA.t.sol

```diff
@@ -137,14 +144,19 @@ contract MerkleVCA_Fuzz_Test is Shared_Fuzz_Test {
     //////////////////////////////////////////////////////////////////////////*/
 
     function expectClaimEvent(LeafData memory leafData) internal override {
-        MerkleVCA.Schedule memory schedule = merkleVCA.getSchedule();
+        // Calculate claim and forgone amount based on the vesting schedule.
+        uint256 claimAmount;
+        uint256 forgoneAmount;
+
+        if (getBlockTimestamp() < merkleVCA.END_TIME()) {
+            uint40 elapsedTime = (getBlockTimestamp() - merkleVCA.START_TIME());
+            uint40 totalTime = merkleVCA.END_TIME() - merkleVCA.START_TIME();
 
-        // Calculate claim amount based on the vesting schedule.
-        uint256 claimAmount = getBlockTimestamp() < schedule.endTime
-            ? (uint256(leafData.amount) * (getBlockTimestamp() - schedule.startTime))
-                / (schedule.endTime - schedule.startTime)
-            : leafData.amount;
-        uint256 forgoneAmount = leafData.amount - claimAmount;
+            claimAmount = (uint256(leafData.amount) * elapsedTime) / totalTime;
+            forgoneAmount = leafData.amount - claimAmount;
+        } else {
+            claimAmount = leafData.amount;
+        }
```

Comment:
Can we move this into a bespoke mirror function somewhere in the tests?

Reply:
Yes!